### PR TITLE
Update every package-lock with npm 7

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Every package-lock are outdated and built with npm 6 while we now use node 14 and npm 7
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Just build every front-end folders, it's testable by a dev
| Possible impacts? | every npm builds

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26021)

### CODE REVIEW

**Review by matks — DISMISSED** [review]

**Review by PierreRambaud — DISMISSED** [review]
As I said it to you, it should be mention in the documentation, and be carefull, the new lock is file no longer backward compatible with old npm versions.

**Comment by NeOMakinG** [comment]
> As I said it to you, it should be mention in the documentation, and be carefull, the new lock is file no longer backward compatible with old npm versions.

We use lockfile v2, which is backward compatible with npm 6 and lockfiles 1

**Review by PierreRambaud — DISMISSED** [review]

**Comment by PierreRambaud** [comment]
> > As I said it to you, it should be mention in the documentation, and be carefull, the new lock is file no longer backward compatible with old npm versions.
> 
> We use lockfile v2, which is backward compatible with npm 6 and lockfiles 1

Yeah, I misstook it with lockfile v3 -_-

**Review by atomiix — DISMISSED** [review]

**Comment by boubkerbribri** [comment]
Waiting for #26118 to test all campaigns.

**Comment by PierreRambaud** [comment]
> Waiting for #26118 to test all campaigns.

The PR has been merged

**Comment by boubkerbribri** [comment]
> > Waiting for #26118 to test all campaigns.
> 
> The PR has been merged

@PierreRambaud Tests started, could you review, please?

**Review by PierreRambaud — DISMISSED** [review]

File: admin-dev/themes/new-theme/package-lock.json [hunk 1]
```diff
@@ -1,19497 +1,8 @@
 {
   "name": "new-theme",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```
Comment (atomiix) [inline admin-dev/themes/new-theme/package-lock.json]: Shouldn't it be `"lockfileVersion": 2`?

**Review by Progi1984 — DISMISSED** [review]

**Review by atomiix — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @NeOMakinG

**Pull request merged**